src/Class Specs/Class_sport.jsx

```diff
@@ -313,18 +291,20 @@ class Sport extends React.Component {
 
     if (found == false) {
       this.Graphs.Team_players(name, this.year).then((result) => {
-        //console.log(result);
         this.teams_player[0].push(`${name}${this.year}`);
-        data = this.format_players(result);
-        this.teams_player[1].push(data);
-        this.Select_team = `${name}`;
-        this.set_co(this.Select_team);
-        console.log("done");
+        this.format_players(result).then((result) => {
+          this.teams_player[1].push(result);
+          this.Select_team = `${name}`;
+          this.set_co(name);
+          console.log("done");
+          if (this.s_c_e == true) {
+            this.search_co_event(name + "1q");
+          }
+        });
       });
-    } else {
-      if (this.s_c_e == true) {
-        this.search_co_event(name);
-      }
+    }
+    if (this.s_c_e == true) {
+      this.search_co_event(name);
```

Comment:
It seems like you remove the if else statement, was their a reason behind that decision. If so, won't this.search_co_event be called twice.

Reply:
That problem was fixed, it was fixed using or saving the same with another string.